README.md

```diff
@@ -45,6 +45,7 @@ If you are using react, wrap your root component with [PersistGate](./docs/Persi
 import { PersistGate } from 'redux-persist/integration/react'
 
 // ... normal setup, create store and persistor, import components etc.
+//example : import { store, persistor } from './configureStore'
```

Comment:
I think this would be clear if you just move `import { store, persistor } from './configureStore'` to right under the PersistGate import above. If you do that I think we can remove line 47, then I'll merge it.

Reply:
Just did that, kindly check.